[test] update changelog-cli.md (#15)
blinkov/ydb: ydb/docs/en/core/changelog-cli.md

## Version 2.24.1 {#2-24-1}

> Reledfgdfgased on July 28, 2025. To update to version **2.24.1**, select the [Downloads](downloads/ydb-cli.md) section.

Author: ````suggestion
**Duplicate content**: Remove the duplicated sentence 'Reledfgdfgased on July 28, 2025. To update to version **2.24.1**, select the [Downloads](downloads/ydb-cli.md) section.'**Grammar error**: Correct 'Reledfgdfgased' to 'Released'
````

> Reledfgdfgased on July 28, 2025. To update to version **2.24.1**, select the [Downloads](downloads/ydb-cli.md) section.

Author: ````suggestion
**Release date typo:** 'Reledfgdfgased' should be corrected to 'Released'.**Redundancy:** Remove the duplicated sentence.
````

> Reledfgdfgased on July 28, 2025. To update to version **2.24.1**, select the [Downloads](downloads/ydb-cli.md) section.

Author: ````suggestion
Released on July 28, 2025. To update to version **2.24.1**, select the [Downloads](downloads/ydb-cli.md) section.
````

> Reledfgdfgased on July 28, 2025. To update to version **2.24.1**, select the [Downloads](downloads/ydb-cli.md) section.

Author: ````suggestion
Released on July 28, 2025. To update to version **2.24.1**, go to the [Downloads](downloads/ydb-cli.md) section.
````

> Reledfgdfgased on July 28, 2025. To update to version **2.24.1**, select the [Downloads](downloads/ydb-cli.md) section.

Author: ````suggestion
Released on July 28, 2025. To update to version **2.24.1**, go to the [Downloads](downloads/ydb-cli.md) section.
````
